src/components/app-configure/chain-management/AppChainManagementForm.vue

```diff
@@ -116,12 +116,11 @@ const chainIDs = chainManagementStore.chainIDs
 async function fetchChainIdUsingRPCUrl(rpcURL: string) {
   try {
     showLoaderRPCValidation.value = true
-    const {
-      data: { result },
-    } = await getChainIDUsingRPCUrl(rpcURL)
-    formData.value.chainId = `${parseInt(result, 16)}`
+    const chainId = await getChainIDUsingRPCUrl(rpcURL)
+    formData.value.chainId = `${chainId}`
```

Comment:
Why not simply use `chainId.toString()` instead of relying on implicit coercion.